bring-it-up/bring-it-up
# Add navbar

## Description
#### Summary:
Created the navigation bar and pages (includes Home, About, Services and Sign In)

#### Changes:
- create About.js, Home.js, Services.js, Navigation.js in component folder
- ex) - make headerComponent.js file in component folder
- ex) - add headerComponent in app.js file

#### How to test this PR:
1) run localhost
2) check that Home, About, Services are on the left side and the link changes when the respective pages are clicked and the name appears as a header on the page
3) check that Sign In is on the right and that the link changes to end in "/signin"
